README.md

```diff
@@ -28,8 +28,11 @@ with tfe.protocol.Pond(server0, server1, crypto_producer) as prot:
     # get input from inputters as private values
     inputs = [prot.define_private_input(inputter) for inputter in inputters]
 
-    # securely sum all inputs and divide by count
-    result = reduce(lambda x,y: x+y, inputs) / len(inputs)
+    # get count inverse as a public value
+      cnt_inv = prot.define_public_variable(np.array([1 / len(inputs)]))
```

Comment:
easy way: `reduce(lambda x, y: x + y, inputs) * (1 / len(inputs))`
best way: add support for public division to pond

Reply:
jup, that works for me -- is that what's done in the actual code..?

Reply:
In this PR, I currently have
```
# get count inverse as a public value
      cnt_inv = prot.define_public_variable(np.array([1 / len(inputs)]))

    # sum all inputs and multiply by count inverse
      result = reduce(lambda x, y: x + y, inputs) * cnt_inv
```

in the meantime I implement public division in pond, do you prefer this or `reduce(lambda x, y: x + y, inputs) * (1 / len(inputs))`